### DISCUSSION TITLE
Is 1.21 planned

### DISCUSSION THREAD

**Discussion by HatTrkPatrk** [RakambdaOrg/FallingTree#691] [General]
Hi, I'm making preparations to setup a 1.21 SMP server for someone and I'm going through their requested modlist to see if support is planned.

Not asking per se for a release date/timetable but I do want to ask if a 1.21 version of FallingTree is at least planned. Thank you! (And sorry for being That Guy™ asking about versions lol)

**Comment by Rakambda** [comment]
There will be a version for it yeah. Usually I make it within a week after the release/modloader is available.
